cartographer/mapping/internal/2d/scan_matching/occupied_space_cost_function_2d.h

```diff
@@ -29,19 +29,18 @@ namespace cartographer {
 namespace mapping {
 namespace scan_matching {
 
-// Computes a cost for matching the 'point_cloud' to the 'probability_grid' with
+// Computes a cost for matching the 'point_cloud' to the 'grid' with
 // a 'pose'. The cost increases when points fall into less occupied space, i.e.
 // at pixels with lower values.
```

Comment:
Should lower become higher here in this comment since it is a correspondence cost?

Reply:
Clarified the comment.